public/javascripts/SVValidate/src/mission/MissionContainer.js

```diff
@@ -66,6 +66,7 @@ function MissionContainer () {
             labelTypeId : missionMetadata.label_type_id,
             missionId : missionMetadata.mission_id,
             missionType : missionMetadata.mission_type,
+            missionsCompleted : svv.missionsCompleted,
```

Comment:
Do you still need the lines you added in this file?

Reply:
No, that is no longer needed. I'm not sure why I wanted to store missionsCompleted inside of a mission object when that mission object can anyways access the global svv.missionsCompleted